UI/CMakeLists.txt

```diff
@@ -444,6 +464,33 @@ elseif(OS_POSIX)
   if(OS_FREEBSD)
     target_link_libraries(obs PRIVATE procstat)
   endif()
+
+  if(OS_LINUX AND ENABLE_WHATSNEW_LINUX)
```

Comment:
Accordingly just check for `ENABLE_WHATSNEW` here.

Reply:
I would keep `OS_LINUX` here to avoid issues with FreeBSD at least.